client-app/shared/account/composables/useUser.ts

```diff
@@ -322,9 +334,51 @@ export function useUser() {
     }
   }
 
+  async function switchOrganization(organizationId: string): Promise<void> {
+    loading.value = true;
+
+    try {
+      await refresh(organizationId);
+
+      broadcast.emit(reloadAndOpenMainPage, null, TabsType.ALL);
+
+      /*const { error, data } = await useFetch("/connect/token")
+        .post(
+          new URLSearchParams({
+            grant_type: "switch_organization",
+            user_id: globals.userId,
+            organization_id: organizationId,
+          }),
+          "application/x-www-form-urlencoded",
+        )
+        .json<ConnectTokenResponseType>();
+
+      if (data.value && !error.value) {
+        const { access_token, token_type, expires_in } = data.value;
+
+        setAccessToken(access_token);
+        setExpiresAt(expires_in);
+        setTokenType(token_type);
+
+        setTimeout(() => {
+          broadcast.emit(reloadAndOpenMainPage, null, TabsType.ALL);
+        }, 1000);
+      } else {
+        Logger.error(switchOrganization.name, error.value);
+      }*/
+    } catch (e) {
+      Logger.error(switchOrganization.name, e);
+    } finally {
+      loading.value = false;
+    }
+  }
+
   return {
     isAuthenticated,
     isCorporateMember,
+    isMultiOrganization: computed(
+      () => user.value?.contact?.organizations?.items && user.value?.contact?.organizations?.items?.length > 1,
```

Comment:
could be just `user.value?.contact?.organizations?.items?.length > 1` , since you use optional chaining

Reply:
No, `items?.length` is possibly undefined.